lib/std/crypto/tlcsprng.zig

```diff
@@ -53,7 +53,7 @@ fn tlsCsprngFill(_: *anyopaque, buffer: []u8) void {
     // std.crypto.random always make an OS syscall, rather than rely on an
     // application implementation of a CSPRNG.
     if (std.options.crypto_always_getrandom) {
-        return defaultRandomSeed(buffer);
+        return std.options.cryptoRandomSeed(buffer);
```

Comment:
Arguably this means the option name should change as well, the use of `getrandom()` is an implementation detail of `cryptoRandomSeed()`'s default implementation. Probably for a follow-up though.